src/applications/personalization/rated-disabilities/sass/rated-disabilities.scss

```diff
@@ -10,13 +10,37 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 .rated-disability-item-container {
     background-color: #f1f1f1;
     margin: 1em 0;
-    padding: 1em;
+    padding: 0.5em 1em 1em;
+
+    &:first-of-type {
+        border-top: 1px solid $color-base;
```

Comment:
We have border classes that can be used to accomplish this [here](https://design.va.gov/utilities/border)